docs/administration/upgrading/guide/automate-upgrades.md

```diff
@@ -120,147 +77,105 @@ Write-Host "Maintenance's response: $maintenanceResponse"
 
 ```PowerShell
 Set-Location "${env:ProgramFiles}\Octopus Deploy\Octopus" 
-
-& .\octopus.server.exe node --instance="OctopusServer" --drain=true --wait=0
-& .\octopus.server.exe service --instance="OctopusServer" --stop
+& .\Octopus.Server.exe node --instance="OctopusServer" --drain=true --wait=0
+& .\Octopus.Server.exe service --instance="OctopusServer" --stop
 ```
 ### Backup the SQL Server database
 
 The simplest backup possible is a full database backup.  Execute the below T-SQL command to save a backup to a NAS or file share:
 
-```
+```sql
 BACKUP DATABASE [OctopusDeploy]
-          TO DISK = '\\SomeServer\SomeDrive\OctopusDeploy.bak'
-             WITH FORMAT;
+  TO DISK = '\\SomeServer\SomeDrive\OctopusDeploy.bak'
+      WITH FORMAT;
 ```
 
 The `BACKUP DATABASE` T-SQL command has dozens of options.  Please refer to [Microsoft's documentation](https://docs.microsoft.com/en-us/sql/relational-databases/backup-restore/create-a-full-database-backup-sql-server?view=sql-server-ver15) or consult a DBA to understand which options you should use.
 
-### Installing the MSI
-
-The Octopus Deploy MSI installation can be automated by calling `msiexec.exe` directly or using a third-party tool such as Chocolatey.
-
-:::warning
-Please run `msiexec.exe` as an administrator.  It is performing an installation, and you will be prompted by Windows to confirm.
-:::
-
-#### Using msiexec.exe
+You can invoke the `BACKUP DATABASE` command from PowerShell using [sqlcmd](https://docs.microsoft.com/en-us/sql/tools/sqlcmd-utility)
 
-Windows includes [msiexec.exe](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec) as part of the base installation.
+### Installing the MSI
 
-The `msiexec.exe` can be invoked via PowerShell by using the Start-Process command.  This example will send in the `/i` and `/quiet` arguments to install the MSI without user interaction:
+Use the downloaded installer by invoking `msiexec.exe`:
 
 ```PowerShell
-$msi = "C:\Temp\OctopusDeploy.2020.4.2.msi"
-Write-Output "Installing MSI $msi" 
-$msiExitCode = (Start-Process -FilePath "msiexec.exe" -ArgumentList "/i $msi /quiet" -Wait -Passthru).ExitCode 
-Write-Output "Server MSI installer returned exit code $msiExitCode" 
+msiexec /i "${env:TEMP}\$msiFilename" /quiet | Out-Null
 ```
 
-#### Using Chocolatey
```

Comment:
It's nice to know that we can also install Octopus using Chocolatey, but we've already downloaded the installer in one of the previous steps. We might as well use that thing here and avoid downloading Octopus twice. I made mention of Chocolatey as an option below but did not give specific instructions because I thought it was just a bit of a distraction.